### PR TITLE
docs: Make clear when rules are removed

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

For all of the removed rules, I added an "Important" block around the removal notice to make it more obvious.

### CODE REVIEW

**Review by mdjermanovic — COMMENTED** [review]
Missing updates for [`space-after-keywords`](https://deploy-preview-17728--docs-eslint.netlify.app/rules/space-after-keywords) and [`space-unary-word-ops`](https://deploy-preview-17728--docs-eslint.netlify.app/rules/space-unary-word-ops).

**Comment by nzakas** [comment]
Thanks, updated.

File: docs/src/rules/space-unary-word-ops.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ title: space-unary-word-ops
 
 Requires spaces after unary word operators.
 
-(removed) This rule was **removed** in ESLint v0.10.0 and **replaced** by the [space-unary-ops](space-unary-ops) rule.
+:::important
+This rule was removed in ESLint v0.10.0 and replaced by the [space-unary-ops](space-unary-ops) rule.
+:::
```
Comment (mdjermanovic) [inline docs/src/rules/space-unary-word-ops.md:11]:
```suggestion
```

To fix the markdown lint error.

**Review by snitin315 — APPROVED** [review]
There's a lint error to fix, otherwise, LGTM

**Review by mdjermanovic — APPROVED** [review]
LGTM when the lint error is fixed.

**Pull request merged**